packages/api/blocks-api/src/blocks/phone-link.block.ts

```diff
@@ -0,0 +1,21 @@
+import { IsString } from "class-validator";
+
+import { BlockData, BlockInput, createBlock, inputToData } from "./block";
+import { BlockField } from "./decorators/field";
+
+class PhoneLinkBlockData extends BlockData {
+    @BlockField()
+    phone: string;
+}
+
+class PhoneLinkBlockInput extends BlockInput {
+    @IsString()
+    @BlockField()
+    phone: string;
```

Comment:
Please add the phone number validation from [IsLinkTarget](https://github.com/vivid-planet/comet/blob/main/packages/api/blocks-api/src/blocks/externalLinkBlock/is-link-target.validator.ts#L32) here